client/components/orderable-list/index.js

```diff
@@ -1,126 +0,0 @@
-/** @format */
```

Comment:
I just renamed this to `PaymentMethodsList` and removed the `@dnd-kit` library dependencies.
Unfortunately it is being picked up as deleted. See `client/components/payment-methods-list/index.js`